src/helpers/calculate.ratings.percentages.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * A fuction to calculate the percentage of ratings and to
+ * generate a structured object of data
+ * @param {Array} data - The array of the data to compute
+ * @returns {Array} data - The computed array
+ */
+export default (data) => {
+  // constructing the array to look like excpected
+  data = data.map(rating => ({
+    rating: rating.dataValues.rating,
+    count: parseInt(rating.dataValues.count, 10)
+  }));
+
+  // initializing variables
+  const ZERO = 0,
```

Comment:
@nkpremices remember - descriptive variable names

Reply:
Oh.. I missed this one. 
I'll do that @MemunaHaruna